src/openforms/registrations/contrib/objects_api/submission_registration.py

```diff
@@ -91,7 +100,7 @@ def _resolve_documenttype(
             description = options["iot_attachment"]
             url_ref = options.get("informatieobjecttype_attachment", "")
         case _:  # pragma: no cover
-            raise RuntimeError(f"Unhandled field '{field}'.")
+            assert_never(field)
```

Comment:
What is the difference here?In the previous code we would have a RuntimeError, is the result kinda the same here?

Reply:
yes, runtime the result is the same, `assert_never` is more typechecker friendly